asr1k_neutron_l3/extensions/asr1koperations.py

```diff
@@ -196,6 +199,16 @@ def update(self, request, id, **kwargs):
         except BaseException as e:
             raise exceptions.HTTPInternalServerError(detail=str(e))
 
+    def sync_data(self, request, id, **kwargs):
+        check_access(request)
+        try:
+            data = self.plugin.get_sync_data(request.context, router_ids=[id])
+            if not data:
+                raise exceptions.HTTPNotFound(detail="No router found for given uuid")
```

Comment:
May be add the UUID for if we want to inspect lock messages

Reply:
came here to say the same, hence the thumbs up, I am not the author ^^

Reply:
It's the response that the user gets when asking for an ID, so they should know it... but can do it for the logs, yeah. Makes sense.